akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -284,7 +285,13 @@ private[akka] class MergeHub[T](perProducerBufferSize: Int) extends GraphStageWi
 
     }
 
-    (logic, Sink.fromGraph(sink))
+    // propagate LogLevels attribute so that MergeHub can be used with onFailure = LogLevels.Off
+    val sinkWithAttributes = inheritedAttributes.get[LogLevels] match {
+      case Some(a) ⇒ Sink.fromGraph(sink).addAttributes(Attributes(a))
```

Comment:
Is that logging from inside of GraphInterpreter which will be affected or what else?

Reply:
yes, it was to be able to silence the hub when some stream was detatched (I have other more specific logging for that and didn't want double noise). It is used in GraphInterpreter, here https://github.com/akka/akka/pull/23828/files#diff-20ce84dc90488608e4a75c28b5ed0dbdL350